docs/index.md

```diff
@@ -12,27 +12,68 @@ hero:
     - theme: brand
       text: Getting Started
       link: /getting-started
+    - theme: alt
+      text: Widget Collection
+      link: /getting-started
     - theme: alt
       text: View on GitHub
       link: https://github.com/FlowFuse/node-red-dashboard
+---
+
+<script setup>
+    import { ref } from 'vue';
+
+    import HomeExtension from './components/HomeExtension.vue';
+    import RecommendedReading from './components/RecommendedReading.vue';
+    import CopyIcon from './components/icons/CopyIcon.vue';
+
+    const copied = ref(false); 
```

Comment:
Just to validate; `copied` can be `const` as it's the reference to false. Than the value is later updated to `true`. 

Why not just `let copied = false` here?

Reply:
using the `ref` is a way of making it accessible to the HTML/template, eqv of `data () { ... }` in a full vue component definition